### DISCUSSION TITLE
Clustering

### DISCUSSION THREAD

**Discussion by git-matrix** [oven-sh/bun#3414] [Q&A] [answered]
As of ``` node.js ```, We can use ``` cluster ``` module for clustering http server. What alternative is with ``` Bun ```?

**Comment by melroy89** [comment]
Cluster is indeed not implemented yet: https://bun.sh/docs/runtime/nodejs-apis#node-cluster

See also ticket: https://github.com/oven-sh/bun/issues/2428

**Reply by harryqt** [reply, marked as answer]
It is now: https://bun.sh/blog/bun-v1.1.25#node-cluster-support

**Reply by melroy89** [reply]
Yes! Finally, let's go!
